xcm/xcm-simulator/example/src/parachain.rs

```diff
@@ -166,6 +166,12 @@ pub mod mock_msg_queue {
 	#[pallet::getter(fn parachain_id)]
 	pub(super) type ParachainId<T: Config> = StorageValue<_, ParaId, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn received_dmp)]
+	/// A queue of received DMP messages
+	pub(super) type ReceivedDmp<T: Config> =
+		StorageValue<_, Vec<(MultiLocation, Xcm<T::Call>)>, ValueQuery>;
```

Comment:
Does this really need a `MultiLocation` when it's always assumed that DMP messages come from the parent, i.e. the relay chain?

Reply:
Good point, I'll make this change